apine.schema.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#", 
+  "$id": "https://github.com/INCF/apine/blob/master/apine.schema.json", 
+  "type": "object", 
+  "additionalProperties": false, 
+  "properties": {
+    "dataset": {
+      "$id": "/properties/dataset", 
```

Comment:
only my ignorance on this: I suppose "/properties/"  in the "/properties/dataset" or others is standard, but can you point me to a place that describe why this is done this way ? I mean, do we have  a choice of chosing something else than "properties" here ? (sorry if the Q. doesn't make  much sense)

Reply:
Great point - I updated these to be downstream from the main schema `$id` - the `properties/` values were just set by the schema generator I used and I forgot to replace.